doc/source/selections.rst

```diff
@@ -0,0 +1,344 @@
+.. -*- coding: utf-8 -*-
+.. _selections:
+
+=======================
+Atom selection language
+=======================
+
+AtomGroups can be created by selecting atoms using the MDAnalysis atom selection language:
+
+  >>> kalp = u.select_atoms("segid KALP")
+
+
+The :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` method of a
+:class:`~MDAnalysis.core.groups.AtomGroup` or a
+:class:`~MDAnalysis.core.universe.Universe` returns an
+:class:`~MDAnalysis.core.groups.AtomGroup`. Selections always return an
+:class:`~MDAnalysis.core.groups.AtomGroup` with atoms sorted according to their
+index in the topology. This is to ensure that there are not any duplicates,
+which can happen with complicated selections.
```

Comment:
Say here that you start with a number of examples and then explain more later. Otherwise this is a bit much for a newbie.

Reply:
I do like the use of links for anyone wanting to learn more, I think that's really helpful.